lib/drivers/animasDriver.js

```diff
@@ -1644,28 +1644,31 @@ module.exports = function (config) {
 
       }
 
-      var suspend = null;
-      var resume = null;
       for(var s in data.suspendResumeRecords) {
         var suspendresumedatum = data.suspendResumeRecords[s];
 
-        suspend = cfg.builder.makeDeviceEventSuspend()
-          .with_deviceTime(suspendresumedatum.suspendDeviceTime)
-          .with_reason({suspended: 'manual'})
-          .set('index', suspendresumedatum.index);
-        cfg.tzoUtil.fillInUTCInfo(suspend, suspendresumedatum.suspendJsDate);
-        suspend = suspend.done();
-        postrecords.push(suspend);
-
         if(suspendresumedatum.resumeJsDate > suspendresumedatum.suspendJsDate) {
```

Comment:
Please make this `>=` to handle brief suspend/resumes in the same minute. I have one in my data that's coming through like this:
```
    {
        "time": "2016-03-21T18:01:00.000Z",
        "timezoneOffset": -420,
        "clockDriftOffset": 0,
        "conversionOffset": 0,
        "deviceTime": "2016-03-21T11:01:00",
        "deviceId": "IR1285-72-36121-15",
        "type": "deviceEvent",
        "subType": "status",
        "status": "suspended",
        "reason": {
            "suspended": "manual"
        },
        "payload": {
            "logIndices": [
                7
            ]
        },
        "annotations": [
            {
                "code": "status/incomplete-tuple"
            }
        ]
    },
```

But the UI shows a suspend and a resume at `2016-03-21T11:01:00`. So will have `duration: 0` which is a bit of a lie, so also maybe another annotation, so we can say "This suspend was too brief to capture the duration" or similar.